languages/swift/exercises/concept/dictionaries/.meta/design.md

```diff
@@ -0,0 +1,33 @@
+The goal of this exercise is to teach the student the basics of the Swift collection type [dictionary][dictionaries].
+
+## Learning objectives
+
+- Initializing empty dictionaries
+- Initializing sets from dictionary literals
+- Inserting key/value pairs
+- Retrieving values
+- Iterating over a dictionary
+
+## Out of scope
+
+- Hashable protocol
+
+## Concepts
+
+- `dictionaries`: Know how to initialize empty dictionaries, know how to initialize dictionaries from dictionary literals, know how to insert key/value pairs, know how to retrieve values, know how to iterate over a dictionary
+
+## Prerequisites
+
+- `arrays`
```

Comment:
I've been told that for the prerequisites, you should leave out prerequisites of prerequisites, so `basics` should be left out here because it is a prerequisite of `arrays`.